bitnami/wordpress/templates/deployment.yaml

```diff
@@ -140,7 +140,7 @@ spec:
             successThreshold: {{ .Values.livenessProbe.successThreshold }}
             failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
             httpGet:
-              path: /wp-login.php
+              path: /wp-admin/install.php
```

Comment:
No. 
The liveness probe changed here to avoid a pod restart during WP maintenance mode. 
A readiness probe gives an answer: redirect traffic to a pod or not. In this case, it shouldn't be changed.